src/views/Home.vue

```diff
@@ -72,12 +73,12 @@
       this.$store.dispatch(FETCH_TAGS)
     },
     computed: {
-      isAuth () {
-        return this.$store.getters[IS_AUTHENTICATED]
-      },
-      tags () {
-        return this.$store.state.home.tags
-      },
+      ...mapGetters({
+        isAuth: [IS_AUTHENTICATED]
```

Comment:
So, instead of this, should we use the key `isAuthenticated` like this:

```js
...mapGetters([
    'isAuthenticated'
])
```

and on the template:

```html
<li v-if="isAuthenticated" class="nav-item">
   ...
</li>
```